{"uuid": "4d6de280-35fa-4d72-9c74-52cdfb127849", "children": ["3580b0f9-9107-4014-9d6b-f6f498c41949"], "befores": [{"name": "_Class__pytest_setup_class", "status": "broken", "statusDetails": {"message": "AttributeError: 'App' object has no attribute 'driver'\n", "trace": "  File \"e:\\ofg\\ceshiren\\python\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"e:\\ofg\\ceshiren\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"e:\\ofg\\ceshiren\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 788, in call_fixture_func\n    res = next(it)\n  File \"e:\\ofg\\ceshiren\\python\\lib\\site-packages\\_pytest\\python.py\", line 701, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"e:\\ofg\\ceshiren\\python\\lib\\site-packages\\_pytest\\python.py\", line 632, in _call_with_optional_argument\n    func(arg)\n  File \"E:\\ofg\\ceshiren\\app_wework\\testcase\\test_contact.py\", line 10, in setup_class\n    self.main = self.app.start().goto_main()\n  File \"E:\\ofg\\ceshiren\\app_wework\\page\\app.py\", line 9, in start\n    if  self.driver==None:\n"}, "start": 1597740389446, "stop": 1597740389446}], "start": 1597740389446, "stop": 1597740390009}